doc-content/enterprise-only/openshift/environment-managed-deploy-proc.adoc

```diff
@@ -18,7 +18,7 @@ If you want to modify the environment defined by the template file, see <<enviro
 +
 [subs="attributes,verbatim,macros"]
 ----
-oc new-app -f <template-file-name> -p {CENTRAL_CAPITAL_UNDER}_HTTPS_SECRET={CENTRAL_ONEWORD}-app-secret -p KIE_SERVER_HTTPS_SECRET=kieserver-app-secret -p MAVEN_REPO_URL=<maven-repo-url>
+oc new-app -f <template-file-name> -p {CENTRAL_CAPITAL_UNDER}_HTTPS_SECRET={CENTRAL_ONEWORD}-app-secret -p KIE_SERVER_HTTPS_SECRET=kieserver-app-secret -p KIE_SERVER_ROUTER_HTTPS_SECRET=smartrouter-app-secret -p MAVEN_REPO_URL=<maven-repo-url>
```

Comment:
Same here, about the ital.

Reply:
Could not reproduce either of these issues. Let's merge as is and sanity-check as draft.